src/services/role/utils/permissions.js

```diff
@@ -27,16 +27,16 @@ const definePermissions = (env, role, ...permissions) => {
 	if (['opt-out', 'enabled'].includes(Configuration.get(env))) {
 		// set defaul permission
 		Role.update({
-			name: role,
-		},
-		{
+				name: role,
+			},
+			{
 
-			$addToSet: {
-				permissions: {
-					$each: permissions,
+				$addToSet: {
```

Comment:
Codacy found an issue: [Expected indentation of 3 tabs but found 4.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6051638)